### DISCUSSION TITLE
[Feature Request]: Hyper terminal app support

### DISCUSSION THREAD

**Discussion by aaronmeder** [beyondcode/herd-community#369] [Ideas]
### Feature Description

Hey guys

I am using [Hyper](https://hyper.is/) as my terminal app.
Would it be possible to support that in the Shortcuts > Terminal select?

Thanks
Aaron

### Is this feature valuable for other users as well and why?

Would help anyone also using Hyper terminal app to allow them to use the Shortcuts feature with their favorite terminal app. Not a game changer for me personally however.

**Comment by CamKem** [comment]
This would be excellent, I have been hoping for it for a while now & I know many of us use Hyper as our main terminal these days.
@mpociot any chance we can get Hyper terminal integration in the settings?
